content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -0,0 +1,186 @@
+
+### Apply or Remove Variable Sets From Inside a Workspace
+
+To apply a variable set to a specific workspace:
+
+1. Go to the workspace and click the **Variables** tab. The **Variables** page appears, showing all workspace-specific variables and variable sets applied to the workspace.
+
+2. Click **Apply Variable Set**, select the variable set you want to apply to your workspace, and click **Apply variable set**. The new variable set appears in the list and the Terraform will use the variables in the set during the next workspace run.
+
+To remove a variable set from within a workspace:
+
+1. Go to the workspace and click the **Variables** tab. The **Variables** page appears, showing all workspace-specific variables and variable sets applied to the workspace.
+2. Click the ellipses button next to the variable set and select **Remove variable set**.
+3. Click **Remove variable set** in the dialog box. Terraform Cloud removes the variable set from this workspace, but it remains available to other workspaces in the organization.
+
+## Overwrite Variable Sets
+
+You can overwrite variables defined in variable sets within a workspace. For example, you may want to use a different set of provider credentials in a specific workspace.
+
+To overwrite a variable from a variable set, [create a new workspace-specific variable](#workspace-specific-variables) of the same type with the same key. Terraform Cloud marks any variables that you overwrite with a yellow exclamation point and provides a link to the variable that will take precedence during runs.
+
+![An overwritten variable marked with a yellow exclamation point](link)
+
+Variables within a variable set can also automatically overwrite, or are overwritten by, variables with the same key in other variable sets applied to the workspace. Refer to [variable precedence](/docs/cloud/workspaces/variables.html#precedence) for more details.
+
+## Variable Values and Format
+
+The limits, allowable values, and required format are the same for both workspace-specific variables and variable sets.
+
+### Security
+
+Terraform Cloud encrypts all variable values securely using [Vault's transit backend](https://www.vaultproject.io/docs/secrets/transit/index.html) prior to saving them. This ensures that no out-of-band party can read these values without proper authorization. However, [descriptions](#variable-description) are not encrypted, so be careful with the information you save in a variable description.
+
+We also recommend using environment variables when possible to avoid unnecessary credential disclosure. Terraform runs receive the full text of [sensitive](#sensitive-values) Terraform variables and may print the value in logs and state files if the configuration sends the value to an output or a resource parameter. Sentinel mocks downloaded from runs will also contain the sensitive values of Terraform variables. Environment variables can also be included in log files if `TF_LOG` is set to `TRACE`.
```

Comment:
I'm pretty sure this is the model but you might want to get someone to check over this edit to make sure that its still technically correct. 

```suggestion
We also recommend passing credentials to Terraform as environment variables instead of Terraform variables when possible, since Terraform runs receive the full text of all Terraform variable values, including [sensitive](#sensitive-values) ones. It may print the values in logs and state files if the configuration sends the value to an output or a resource parameter. Sentinel mocks downloaded from runs will also contain the sensitive values of Terraform variables. 

Although Terraform cloud does not store environment variables in state, it can include them in log files if `TF_LOG` is set to `TRACE`.
```

Reply:
ohhh thank you! okay let me ask about this one, too. Thank you for the edit!

Reply:
confirmed with two people. I'm going to push this.